vm/vm/src/predeploys/predeploy.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { createTevm } from '../createTevm.js'
+import { DaiContract } from '../test/DaiContract.sol.js'
+import { definePredeploy } from './definePredeploy.js'
+import { Address, hexToBytes, toBytes } from '@ethereumjs/util'
+import { createTevmContract } from '@tevm/contract'
```

Comment:
I am having issues getting the types from `@tevm/contract`

Could not find a declaration file for module '@tevm/contract'. '/Users/matt/dev/tevm-monorepo/packages/contract/dist/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/tevm__contract` if it exists or add a new declaration (.d.ts) file containing `declare module '@tevm/contract';`ts(7016)

Reply:
Looks like this is [breaking builds ](https://github.com/evmts/tevm-monorepo/actions/runs/7303284211/job/19903484080?pr=777)

Reply:
I think this suggestion will likely fix your issue https://github.com/evmts/tevm-monorepo/pull/777#discussion_r1435313311

Reply:
The issue persists with the updated package.json. It is erroring in the CI.

Reply:
Works fine for me and CI. Try running `bun clean && pnpm i && bun allz` from root of repo and if it keeps failing let me know and I'll help debug what is wierd about your environment

Reply:
Actually was able to repro it myself. It' sa wierd bug so I just pushed a workaround rather than debugging it. This package needs to be refereshed in near future anyways to switch to NodeNext and compile contracts which might make issue go away.